.github/workflows/.vs/workflows/v17/DocumentLayout.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Version": 1,
+  "WorkspaceRootPath": "C:\\Users\\Liam_\\source\\repos\\ShaunL12345\\SET09402CW\\.github\\workflows\\",
```

Comment:
Check whether local paths should be commited to repository

Reply:
LJ - This is updated everytime and show not be an issue.